commit/merkleroot/rmn/controller.go

```diff
@@ -174,19 +174,19 @@ func (c *controller) ComputeReportSignatures(
 		}
 	}
 
-	minObserversMap, err := c.rmnHomeReader.GetMinObservers(rmnRemoteCfg.ConfigDigest)
+	homeFMap, err := c.rmnHomeReader.GetF(rmnRemoteCfg.ConfigDigest)
 	if err != nil {
-		return nil, fmt.Errorf("get min observers: %w", err)
+		return nil, fmt.Errorf("get f: %w", err)
 	}
 	// Filter out the lane update requests for chains without enough RMN nodes supporting them.
 	for chain, l := range updatesPerChain {
-		if _, exists := minObserversMap[cciptypes.ChainSelector(chain)]; !exists {
-			return nil, fmt.Errorf("no min observers for chain %d", chain)
+		if _, exists := homeFMap[cciptypes.ChainSelector(chain)]; !exists {
+			return nil, fmt.Errorf("no f for chain %d", chain)
 		}
```

Comment:
Maybe pull this out of the if statement to avoid the double lookup and noisier code:

```suggestion
		chainF, exists := homeFMap[cciptypes.ChainSelector(chain)]
		if !exists { return nil, fmt.Errorf("no F for chain %d", chain) }
		// use chainF below
```